Sky Atlas/Sky Atlas.html

```diff
@@ -3960,6 +3959,304 @@ <h1>Sections & Primary Docs</h1>
+        <tr>
+          <td>
+            <dfn>A.1.9
+              - Sky Core Governance Security - Emergency Spells - Protego</dfn>
+          </td>
+          <td>Sky Core Governance Security - Emergency Spells - Protego</td>
+          <td>Core</td>
+          <td>Protego is a contract that allows Sky Governance to cancel the execution of planned governance actions that are awaiting the expiration of the Governance Security Module Pause Delay (GSM Pause Delay). See <dfn>A.1.9 - Sky Core Governance Security - Governance Security Delay Requirements - Pause Delay</dfn>. The subdocuments herein specify the features of the Protego contract and its authorized use.</td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>A.1.9
+              - Sky Core Governance Security - Emergency Spells - Protego - Protego Features</dfn>
+          </td>
+          <td>Sky Core Governance Security - Emergency Spells - Protego - Protego Features</td>
+          <td>Core</td>
+          <td>The Protego contract allows canceling the execution of planned governance actions within the Sky Protocol. Technical updates and other protocol changes are implemented as Spells within the Sky Protocol, approved by Sky Governance via Executive Votes. These planned governance actions have a time delay before taking effect. In this period between approval and execution, the Protego contract allows for planned governance actions to be canceled. The documents herein specify the core features of the Protego contract.</td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>A.1.9
+              - Sky Core Governance Security - Emergency Spells - Protego - Protego Features - Emergency Drop Spells</dfn>
+          </td>
+          <td>Sky Core Governance Security - Emergency Spells - Protego - Protego Features - Emergency Drop Spells</td>
+          <td>Core</td>
+          <td>The Protego contract can be used to deploy an Emergency Drop Spell. Once authorized by Sky Governance, the Emergency Drop Spell can cancel any planned governance action that is awaiting GSM Pause Delay expiration.</td>
+        </tr>
+        <tr>
+          <td>
+            <dfn>A.1.9
+              - Sky Core Governance Security - Emergency Spells - Protego - Protego Features - Emergency Drop Spells - Deployment Of Emergency Drop Spells</dfn>
+          </td>
+          <td>Sky Core Governance Security - Emergency Spells - Protego - Protego Features - Emergency Drop Spells - Deployment Of Emergency Drop Spells</td>
+          <td>Core</td>
+          <td>Any user can permissionlessly deploy an Emergency Drop Spell by invoking the deploy function on the Protego contract with parameters identifying the planned governance action to be canceled:<br><code>deploy(address _usr, bytes32 _tag, bytes memory _fax, uint256 _eta)(address)</code><br>The parameters of the deploy function are defined in <dfn>A.1.9 - Sky Core Governance Security - Emergency Spells - Protego Parameters</dfn>.</td>
```

Comment:
Let's use `<p>` tags rather than `br` tags here, it will improve the rendering.

![image](https://github.com/user-attachments/assets/98f3d1db-65bf-4ec0-a001-d66bc43ee47c)

vs

![image](https://github.com/user-attachments/assets/ba5ef601-63ba-43ae-a19a-e81a0b46e813)
```suggestion
          <td><p>Any user can permissionlessly deploy an Emergency Drop Spell by invoking the deploy function on the Protego contract with parameters identifying the planned governance action to be canceled:</p><p><code>deploy(address _usr, bytes32 _tag, bytes memory _fax, uint256 _eta)(address)</code></p><p>The parameters of the deploy function are defined in <dfn>A.1.9 - Sky Core Governance Security - Emergency Spells - Protego Parameters</dfn>.</p></td>
```